sdk/rust/src/contract/contract_function_parameters.rs

```diff
@@ -0,0 +1,1268 @@
+    #[allow(dead_code)]
+    pub fn add_uint160_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint160[]", 20)
+    }
+
+    /// Add a `uint168[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint168_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint168[]", 21)
+    }
+
+    /// Add a `uint176[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint176_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint176[]", 22)
+    }
+
+    /// Add a `uint184[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint184_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint184[]", 23)
+    }
+
+    /// Add a `uint192[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint192_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint192[]", 24)
+    }
+
+    /// Add a `uint200[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint200_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint200[]", 25)
+    }
+
+    /// Add a `uint208[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint208_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint208[]", 26)
+    }
+
+    /// Add a `uint216[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint216_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint216[]", 27)
+    }
+
+    /// Add a `uint224[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint224_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint224[]", 28)
+    }
+
+    /// Add a `uint232[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint232_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint232[]", 29)
+    }
+
+    /// Add a `uint240[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint240_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint240[]", 30)
+    }
+
+    /// Add a `uint248[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint248_array(&mut self, val_array: &[BigUint]) -> &mut Self {
+        self.add_int_array(val_array, "uint248[]", 31)
+    }
+
+    /// Add a `uint256[]` argument to the `ContractFunctionParameters`
+    #[allow(dead_code)]
+    pub fn add_uint256_array(&mut self, val_array: &[BigUint]) -> &mut Self {
```

Comment:
```suggestion
    pub fn add_uint256_array(&mut self, values: &[BigUint]) -> &mut Self {
```
(and so on for the rest)